tensorflow/compiler/xla/pjrt/gpu/se_gpu_pjrt_client.cc

```diff
@@ -38,6 +38,7 @@ limitations under the License.
 
 #ifdef TENSORFLOW_USE_ROCM
 #include "rocm/rocm_config.h"
+#include "tensorflow/compiler/xla/pjrt/gpu/nccl_id_store.h"
```

Comment:
Our linter is sad because this file is included twice. Can we hoist both this and the CUDA copy of the include out of the ifdef blocks?

Reply:
I should have caught this in my own review. Missed it!
Anyways, I was about to push a change, but this is now merged.
Should I create another PR that moves the #include outside the #ifdef?